katas/es6/language/__raw-metadata__.js

```diff
@@ -1249,7 +1248,7 @@ export const all = {
             ARROW_FUNCTION_BASICS,
             DEFAULT_PARAMETERS_BASICS,
           ],
-          publishDateUTC: toUtcDate(2015, date.JULY, 14, 9, 29),
+          publishDateUTC: date.toUtcDate(2015, date.JULY, 14, 9, 29),
```

Comment:
hehe, i just moved everything to us `toUtcDate` in here, to have less noise in this file, this would revert that